code/game/objects/items/storage/medkit.dm

```diff
@@ -393,17 +393,17 @@
 	///if you add a new one don't forget to update /datum/crafting_recipe/medbot/on_craft_completion()
 	var/obj/item/bot_assembly/medbot/medbot_assembly = new
 	if (istype(src, /obj/item/storage/medkit/fire))
-		medbot_assembly.set_skin("ointment")
+		medbot_assembly.set_skin("medibot_burn")
 	else if (istype(src, /obj/item/storage/medkit/toxin))
-		medbot_assembly.set_skin("tox")
+		medbot_assembly.set_skin("medibot_toxin")
 	else if (istype(src, /obj/item/storage/medkit/o2))
-		medbot_assembly.set_skin("o2")
+		medbot_assembly.set_skin("medibot_o2")
 	else if (istype(src, /obj/item/storage/medkit/brute))
-		medbot_assembly.set_skin("brute")
+		medbot_assembly.set_skin("medibot_brute")
 	else if (istype(src, /obj/item/storage/medkit/advanced))
-		medbot_assembly.set_skin("advanced")
+		medbot_assembly.set_skin("medibot_advanced")
 	else if (istype(src, /obj/item/storage/medkit/tactical))
-		medbot_assembly.set_skin("bezerk")
+		medbot_assembly.set_skin("medibot_bezerk")
```

Comment:
aw hell naw bruh what the fuck is this `else if` chain chicanery. in an ideal world the system would look like this

```dm
/// Returns the skin we want the medbot assembly to assume
/obj/item/storage/medkit/proc/get_applicable_skin()
	return null

/obj/item/storage/medkit/o2/get_applicable_skin()
	return "medibot_o2"
```

and then in this proc we would just do

```dm
var/obj/item/bot_assembly/medbot/medbot_assembly = new
var/applicable_skin = get_applicable_skin()
if(!isnull(applicable_skin))
	medbot_assembly.set_skin(applicable_skin)
```

of course you probably don't have to do that in this PR but it would be very nice because if people want to add more medbot sprites (or if you were thinking of doing that at a later date) then this would make it far more extensible

Reply:
oh yeag i do want to add one (coronerbot) and might do so in the future, the hard part is figuring out how to do any bespoke behaviors without hitting the simple mob freeze